telegram/bot.go

```diff
@@ -0,0 +1,147 @@
+package telegram
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/NicoNex/echotron/v3"
+	"github.com/labstack/gommon/log"
+	"github.com/ngoduykhanh/wireguard-ui/store"
+)
+
+type SendRequestedConfigsToTelegram func(db store.IStore, userid int64) []string
+
+type TgBotInitDependencies struct {
+	DB                             store.IStore
+	SendRequestedConfigsToTelegram SendRequestedConfigsToTelegram
+}
+
+var (
+	TelegramToken            string
+	TelegramAllowConfRequest bool
+	TelegramFloodWait        int
```

Comment:
```suggestion
	Token            string
	AllowConfRequest bool
	FloodWait        int
```

The variable name shouldn't start with package name.